### ISSUE TITLE
Reproduce results on CARS dataset.

### ISSUE THREAD

**Issue by shijianjian** [dvl-tum/group_loss#3]
Hi,

Thanks for the great repo which is pretty easy to read and run.

Whilst I was trying to reproduce the results for the CARS dataset, I found it is quite away from the reported. Specifically, I got something (for training data) around:
```bash
INFO:root:NMI: 70.262
INFO:root:R@1 : 72.315
INFO:root:R@2 : 82.956
INFO:root:R@4 : 90.764
INFO:root:R@8 : 95.123
```

For the simplicity, I only went for ```python train.py``` since the performance shall be similar between pretrained and non-pretrained models as suggested in the ```README.md``` file. The dataset used is the training data from "https://ai.stanford.edu/~jkrause/cars/car_dataset.html", and all other param used are the defaults from ```train.py```.

I wondering if there is anything I missed?

Thank you in advance for your kind help.

Best,
Jian

**Comment by TheRevanchist** [comment]
Can you please try with the pretrained versions (I have included them in the code)?

Also, please can you check for the PyTorch version? I had used back then PyTorch 1.2 and torchvision 0.2.

Best,
Ismail

**Comment by torki-hossein** [comment]
Hi, my result with pre-trained model (bn_inception) is same with reported on paper:
R@1:85.02
R@8:96.98
embedding dim=1024
loss=loss1+loss2
#loss1=group_loss
#loss2=CE
